tests/std/tests/P1222R4_flat_set/test.cpp

```diff
@@ -288,6 +288,12 @@ void test_extract() {
     assert(ranges::equal(cont, elements));
 }
 
+void test_erase() {
+    flat_set<int> fs{1};
+    fs.erase(1);

```

Comment:
This was not compiling, as `_Erase` got `const key&` which is not same as `key`.